docs/blog/2022-12-19-windows_ssh.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Workaround for Windows SSH "Correupted MAC on input" Error
+---
+Some people who use Windows 10/11 computers to ssh to Eagle from a Windows command prompt, powershell, or via Visual Studio Code's SSH extension might receive an error message about a "Corrupted MAC on input" or "message authentication code incorrect." This error is due to an outdated OpenSSL library included in Windows and a security-mandated change to ssh on Eagle. However, there is a functional workaround for this issue. (Note: If you are not experiencing the above error, you do not need and should not use the following workaround.)
+
+For command-line and Powershell ssh users, adding "-m hmac-sha2-512" to your ssh command will resolve the issue. For example: "ssh -m hmac-sha2-512 <username>@eagle.hpc.nrel.gov"
```

Comment:
The "\<username\>" name part does not get rendered as written (probably because of the "<" and ">" characters.  Using inline code style fixes this (suggest also changing the first usage as well, for consistency):
```suggestion
For command-line and Powershell ssh users, adding `-m hmac-sha2-512` to your ssh command will resolve the issue. For example: `ssh -m hmac-sha2-512 <username>@eagle.hpc.nrel.gov`.
```

Reply:
Fixed, thanks!